hilla-jandex/pom.xml

```diff
@@ -93,4 +93,50 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>distribution</id>
+            <properties>
+                <maven.javadoc.skip>true</maven.javadoc.skip>
+                <maven.source.skip>true</maven.source.skip>
```

Comment:
They are skipped because the module has no sources, so the default plugins will not produce the jar files.
This is the reason `maven-jar-plugin` is used instead to generate the two empty jars.